apps/eclipse/eclipse.talon

```diff
@@ -139,10 +139,10 @@ refactor this: key(alt-shift-i)
 
 #code navigation
 (go declaration | follow): key(f3)
-go back: key(alt-left)
-go forward:  key(alt-right) 
-# go implementation: 
-# go recent: 
+action(user.jump_back): key(alt-left)
+action(user.jump_forward): key(alt-right)
+action(user.go_inside): key(f3)
+action(user.go_bracket): key(ctrl-shift-p)
```

Comment:
These need to be implemented in a python file as talon action definitinos are deprecated